array_analyzer/transform/array_generation.py

```diff
@@ -52,3 +55,116 @@ def build_centroid_binary_blocks(cent_list, image_, params_, return_type='region
         return target*image_
     elif return_type == 'region':
         return target
+
+def coord_to_spot_int(coords, im_int, background, params):
```

Comment:
How about `get_spot_int()` or `get_spot_intensity()`? If we change the data structure to dataframe, the function would extract useful properties from regionprops objects, save them in the dataframe, and return the dataframe rather than passing regionprops objects

Reply:
Either of those is fine.  I think we should leave the data structure as is and adjust that in a later PR